pyrdp/mitm/cli.py

```diff
@@ -145,6 +146,7 @@ def buildArgParser():
                         default="INFO", choices=["INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"])
     parser.add_argument("-F", "--log-filter",
                         help="Only show logs from this logger name (accepts '*' wildcards)", default="")
+    parser.add_argument("--auth", help="Specify allowed authentication mechanisms (Comma-separated, choose from: tls, ssp)", default="tls")
```

Comment:
Maybe not for this PR, but PyRDP supports the standard RDP security as well, so I think it should be included in the options.

Reply:
None is always implicitly supported if the server selects NONE, hence why it's not on that list, and the default is TLS.

That being said, there's a bug (See X224MITM.py:119) where `startTLS` is done according to what the client requests, not what the server replies. I added a **FIXME** label because I didn't want to change the behavior of the code in this PR.

You also made me realize that in `onConnectionRequest` we should not blindly set the support auth methods, but rather, allowing only the configured ones through and setting everything else to false, otherwise if a non-CredSSP client connects, but `PyRDP` is started with `--auth ssp`, the server might mistakenly select SSP, which the client won't be able to respect.